benefits of insight (#164)
DaoCloud/DaoCloud-docs: docs/en/insight/03ProductBrief/benefits.md

# Benefits of Insight

- Multi-cluster Management

    Through the unified storage of metrics, logs and traces data in the Global clusters, the monitoring of multi-cluster can be realized and users can aggregate query of multi-cluster data.

- Out-of-the-box

>     - Dashboard: Insight provides a variety of out-of-the-box prefabricated monitoring dashboards. You can comprehensively monitor clusters, nodes, workloads and other components through built-in the dashboards.

Reviewer: ```suggestion
    - Dashboard: Insight provides a variety of out-of-the-box prefabricated monitoring dashboards. You can comprehensively monitor clusters, nodes, workloads and other components through the built-in dashboards.
```

>     - Alert rules: Insight provides built-in alarm rules that enable you to monitor cluster resources, system components and other basic indicators out of the box without configuration.

Reviewer: ```suggestion
    - Alert rules: Insight provides built-in alarm rules that enable you to monitor cluster resources, system components, and other basic metrics out of the box without configuration.
```

- High Availability

>     - Lightweight agent, which supports one-click install the Helm.

Reviewer: ```suggestion
    - Lightweight agent, which supports for one-click installation with Helm.
```

>     - The data storage component supports multi copies to ensure high availability of data.

Reviewer: ```suggestion
    - The data storage component supports multiple replicas to guarantee the high availability of your data.
```

- Open Source

>     - It is compatible with standard open source Prometheus and supports native PromQL to query metrics.

Reviewer: ```suggestion
    - Compatible with Prometheus and supporting for the native PromQL to query metrics.
```

>     - It is compatible with standard Open Source ` Prometheus Yaml ` to collect configuration file and suitable for customizing the monitoring collection rule ServiceMonitor in Kubernetes.

Reviewer: ```suggestion
    - Compatible with Prometheus YAML to collect profiles and suitable for customizing the `ServiceMonitor` rule in a Kubernetes cluster.
```
